src/browser/provider/built-in/dedicated/chrome/browser-client.ts

```diff
@@ -147,7 +152,7 @@ export class BrowserClient {
         viewportSize.width = newWidth;
         viewportSize.height = newHeight;
 
-        const client = await this.getActiveClient();
+        const client = await this.getActiveClient() as remoteChrome.ProtocolApi;
```

Comment:
Since `this.getActiveClient` can return `null`/`undefined`, shouldn't we check the return value instead of the type conversion?

Reply:
When we call getActiveClient inside the `browser-client.ts` method, I expect that client is not empty. If it is empty for some reason I would prefer to have an exception instead of unexpected behavior. What do you think?